### PR TITLE
get notes comments

### PR DESCRIPTION
**Pull request by VZFV**

### CODE REVIEW

**Review by sue627 — APPROVED** [review]
image
image

reviewed and tested. the code is looks good to me, and the test result is works.

File: backend/source/apis/notes/get_notes_comments.go [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+				FROM comments
+				WHERE note_id = ?`
+
+	rows, err := db.Raw(rawSQL, noteid).Rows()
+	if err != nil {
+		errResp = commonutils.GenerateErrResp(http.StatusInternalServerError, err.Error())
+		return
+	}
+	comments := models.CommentObject{}
+	for rows.Next() {
```
Comment (shp076) [inline backend/source/apis/notes/get_notes_comments.go:60]: Move line 59 inside the loop can fix bugs.

**Review by shp076 — CHANGES_REQUESTED** [review]
Bugs with comments resp. Return same contents.
DB:
image

Resp: 
image

File: backend/source/apis/notes/get_notes_comments.go [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+	noteid := params.NoteID
+
+	resp = &models.NoteCommentsResponse{
+		Comments: []*models.CommentObject{},
+	}
+
+	rawSQL := `SELECT content, note_ID, username 
+				FROM comments
+				WHERE note_id = ?`
```
Comment (shp076) [inline backend/source/apis/notes/get_notes_comments.go:52]: Also return `comment_id`, it can be used in `DELETE /v1/comments/{id}`

**Review by shp076 — CHANGES_REQUESTED** [review]

**Review by shp076 — APPROVED** [review]
Looks good

**Pull request merged**